tests/MigratorTest.php

```diff
@@ -2,16 +2,44 @@
 
 namespace TomSchlick\RequestMigrations\Tests;
 
+use Illuminate\Support\Facades\Event;
+use TomSchlick\RequestMigrations\Events\RequestHasMigrated;
+use TomSchlick\RequestMigrations\Events\RequestIsMigrating;
+use TomSchlick\RequestMigrations\Events\ResponseHasMigrated;
+use TomSchlick\RequestMigrations\Events\ResponseIsMigrating;
+
 class MigratorTest extends TestCase
 {
     public function setUp()
     {
         parent::setUp();
+
+        Event::fake();
     }
 
     /** @test */
     public function it_will_only_get_versions_behind_current()
     {
         $this->markTestIncomplete();
     }
+
+    protected function assertMigrationEventsFired()
```

Comment:
Future use for testing I assume?

Reply:
Yeah. I didn't have the brain power to write more tests last night to generate a fake request / response.

Reply:
Cool! Totally makes sense!